examples/plasticity/WellboreProblem_PoroElastic.xml

```diff
@@ -0,0 +1,281 @@
+<!--
```

Comment:
You may consider defining a common WellboreProblem_Base.xml file, and then two shorter XML files that include the common base specification to derive a plastic and elastic version.

Reply:
Good suggestion, base file will be prepared